src/auth/computeclient.ts

```diff
@@ -33,8 +33,8 @@ export class Compute extends OAuth2Client {
    * Retrieve access token from the metadata server.
    * See: https://developers.google.com/compute/docs/authentication
    */
-  constructor() {
-    super();
+  constructor(refreshTokenEarlyMillis?: number) {
```

Comment:
Can you make this an options object rather than a positional parameter? Similar to JWT.

Reply:
Done -- also did the same for UserRefreshClient.